Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2185,7 +2186,7 @@ def condition(self, fragment, stepList, key, hasHarvest):
 
 class UpgradeWorkflow_DDDDB(UpgradeWorkflow):
     def setup_(self, step, stepName, stepDict, k, properties):
-        if 'Run3' in stepDict[step][k]['--era'] and 'Fast' not in stepDict[step][k]['--era']:
+        if 'Run3' in stepDict[step][k]['--era']  and '2023' not in stepDict[step][k]['--era'] and 'Fast' not in stepDict[step][k]['--era']:
```

Comment:
(Just to understand) What is the purpose of this modification?
Should we add 2024, 2025, etc. here in the coming years?

Reply:
This is just a temporary fix to not screw up on modification. The follow up PR should clean this thing out when we have a clear Era, and all necessary Era(s) are in place, i.e. DDD for 2023,4,5 or FastSim for 2023,4,5.